programs/dynamic-bonding-curve/src/instructions/initialize_pool/ix_initialize_virtual_pool_with_spl_token.rs

```diff
@@ -178,7 +182,12 @@ pub fn handle_initialize_virtual_pool_with_spl_token<'c: 'info, 'info>(
         initial_base_supply,
     )?;
 
-    // remove mint authority
+    // update mint authority
+    let new_mint_authority = match token_authority {
```

Comment:
Impl function in TokenAuthorityOption: `get_mint_authority(creator: Pubkey, partner: Pubkey) -> Option<Pubkey> {}`

Reply:
implemented